gatekeeper-3.17.yaml

```
@@ -59,6 +59,15 @@ subpackages:
       provides:
         - gatekeeper-gator=${{package.full-version}}
 
+  - name: ${{package.name}}-crds
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.destdir}}/crds
+          cp charts/gatekeeper/crds/* ${{targets.destdir}}/crds
```

```suggestion
          mkdir -p ${{targets.contextdirdir}}/crds
          cp charts/gatekeeper/crds/* ${{targets.contextdir}}/crds
```
given this is a subpackage we can consider copying things over here else this package is coming out empty.